package.json

```diff
@@ -66,7 +66,7 @@
     "simplify-js": "^1.2.1"
   },
   "devDependencies": {
-    "@mapd/connector": "git://github.com/mapd/mapd-connector.git#67dd489",
+    "@mapd/connector": "git://github.com/mapd/mapd-connector.git#ce39eb2",
```

Comment:
The tip of master currently is 89683a6. Just make sure you this one here works for you.